### PR TITLE
document PR process

### PR DESCRIPTION
**Pull request by flyinprogrammer**
Oy, this time, really only my changes. Apparently I need more coffee.

### CODE REVIEW

**Review by OlegDokuka — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -58,6 +66,22 @@ npx cross-env \
...
+The `npm run deploy` command will publish the static site into the `./docs` directory, you'll then need to commit
+it so that the site goes live:
+
+```shell script
+git add --all
+git commit -m "your change"
+git push origin $OUR_BRANCH_NAME
+```
+
+We use the `docs` folder instead of a branch because long ago we made that decision. If it serves you better to do a
```
Comment (viglucci) [inline CONTRIBUTING.md:85]: > then you should be familiar with comfortable with Node JS, Docusaurus

😂 It is the way it is because of the way it is.

Comment (flyinprogrammer) [reply CONTRIBUTING.md:85]: I mean basically :joy: I was trying to let folks know that a choice had to be made, we made it long ago, we haven't changed, but there's no "we know something you don't know" about how to do these things. the joy of software!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,38 +18,46 @@ Fork using the command line:
 
 ### Make your changes
 
-Most changes will be expected to be documentation updates in the [markdown docs](/content-docs). If you are changing more than the Markdown documentation pages, then you should be familiar with comfortable with Node JS, Docusaurus, and ReactJS.
+Most changes will be expected to be documentation updates in the [markdown docs](/content-docs). If you are changing
+more than the Markdown documentation pages, then you should be familiar with comfortable with Node JS, Docusaurus,
```
[the remark this answers is not in the document]
Comment (flyinprogrammer) [reply CONTRIBUTING.md]: I think this is addressed now.

**Review by viglucci — APPROVED** [review]

**Pull request merged**